ansible/roles/products/teleop/server/desktop-icons/tasks/demohands-unimanual.yml

```diff
@@ -10,6 +10,7 @@
     ethercat_interface_value: "{{ ethercat_interface }}"
     hand_serial_value: "634"
     hand_side_value: "right"
+    mapping_path_value: "$(find sr_edc_launch)/mappings/default_mappings/rh_E_v3.yaml"
```

Comment:
Shouldn't those be v4?

Reply:
no, those are demohands A, B, C, D which are still using the v3 mapping as far as I know